common/source/docs/common-object-avoidance-landing-page.rst

```diff
@@ -24,20 +25,18 @@ Avoidance Features
     Simple Object Avoidance <common-simple-object-avoidance>
     Autonomous Mission Re-routing via Bendy Ruler <common-oa-bendyruler>
```

Comment:
An existing issue so not a blocker for this PR but at some point I'd like to change these to "BendyRuler path planning around obstacles" and "Dijkstra's path planning around obstacles"

Reply:
might as well do it now...